### PR TITLE
Mr android back issue

### PR DESCRIPTION
**Pull request by masterrom**
Android back button exits the app only when the current screen is the portfolio screen or pin page. The back button also goes back to the previous screen.

### CODE REVIEW

**Comment by GregTheGreek** [comment]
@masterrom Please re-do your commit message to say `Closes #282` in order to properly reference and issue.

To do so you can just use `git commit --amend` and fix the comit message, you may have to `push --froce`

File: src/navigation/stack/index.js [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
+  constructor(props) {
+    super(props)
+    BackHandler.addEventListener("hardwareBackPress", this.onBackPress  );
+  }
+
+  onBackPress = () => {
+    const { dispatch } = this.props;
+    if (this.props.navigationState.index === 2) {
+      return false;
+    }else if (this.props.navigationState.index == 1) {
```
Comment (CFDevelopment) [inline src/navigation/stack/index.js]: Spacing on else if. ES lint

File: src/navigation/stack/mainBucketStack.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { StackNavigator, DrawerNavigator, DrawerItems } from 'react-navigation';
-import { View, StyleSheet } from 'react-native';
+import { View, StyleSheet, BackHandler } from 'react-native';
```
Comment (CFDevelopment) [inline src/navigation/stack/mainBucketStack.js]: BackHandler is defined and never used. Is it needed?

File: src/screens/main/portfolio/index.js [hunk 3]
```diff
@@ -206,6 +206,13 @@ class Portfolio extends Component {
    */
 
   render() {
+
```
Comment (CFDevelopment) [inline src/screens/main/portfolio/index.js]: Remove log and white space

File: src/navigation/stack/index.js (hunk 1 above)
Comment (masterrom) [reply src/navigation/stack/index.js]: Done

File: src/navigation/stack/mainBucketStack.js (hunk 2 above)
Comment (masterrom) [reply src/navigation/stack/mainBucketStack.js]: its not needed here.

File: src/screens/main/portfolio/index.js (hunk 3 above)
Comment (masterrom) [reply src/screens/main/portfolio/index.js]: Done

**Pull request merged**